slides.md

```diff
@@ -16,10 +16,13 @@ mdc: true
 
 ---
 
-## Presentation title
+## Enabling UEFI Secure Boot in XCP-ng: Establishing a Robust Chain of Trust
```

Comment:
Any idea how it's supposed to work in this repo?
We should add slidev presentations here, ore replace offline? Both ways seem weird to do.

Reply:
I don't know sli.dev very well. It could be managed incorrectly, or there may be work upstream to manage scenarios like ours. This is something to check.

Definitely adding all presentations will kill performance and cause issue for speaker.

Most likely, in configuration, we use `slides.md`, which should not be committed to the repository but remains a template. It is modified when you know what you want to render. I guess that could be scripted.

Reply:
`slides.md` is the default file name, you can pass a different one (it's confusingly called "entry" in parameters).  Then contents of this file might need to be repeated in the actual presentation file with correct relative paths unless there is some way to reduce duplication.

Reply:
Yeah:

```
Positionals:
  entry  path to the slides markdown entry       [string] [default: "slides.md"]
```

I'm trying to make it work, but it can't find the theme directory or intro.png, it also doesn't seem to see public either on `..` or `../..`, so I'm not sure how it is supposed to work.